code/game/objects/items/_item_materials.dm

```diff
@@ -113,18 +88,24 @@
 			obj_flags |= OBJ_FLAG_CONDUCTIBLE
 		else
 			obj_flags &= (~OBJ_FLAG_CONDUCTIBLE)
-		update_force()
-		if(material_alteration & MAT_FLAG_ALTERATION_NAME)
-			SetName("[material.adjective_name] [initial(name)]")
+		update_attack_force()
+		update_name()
 		if(material_armor_multiplier)
 			armor = material.get_armor(material_armor_multiplier)
 			armor_degradation_speed = material.armor_degradation_speed
 			if(length(armor))
 				set_extension(src, armor_type, armor, armor_degradation_speed)
 			else
 				remove_extension(src, armor_type)
+
 	queue_icon_update()
 
+/obj/item/proc/update_name()
+	if(material_alteration & MAT_FLAG_ALTERATION_NAME)
+		SetName("[material.adjective_name] [initial(name)]")
+	else
+		SetName(initial(name))
```

Comment:
this causes growns to be named produce, should probably fix that. this will probably also lead to a bunch of other bugs... i am really not looking forward to having to dig through these